### ISSUE TITLE
Not Enough Space

### ISSUE THREAD

**Issue by Astniks** [TitansBane/Elysium-Remastered#58]
Hi, i just got a fresh install on my PC of Windows 10 and I have around 480 GB of free space or so. After downloading this pack and following all the instructions for 3 days straight I find that this mod pack went through all my space and is stuck on step 13 extracting THE last archive. I'm afraid I'm stuck. Is it possible to delete the downloaded mods to free up some space after extracting them? or are they crucial to the automated process of installation?

Please suggest and advise a solution. Thank you for the fantastic mod pack!

P.S. I'm not very skilled with modding and no I cannot buy 1TB of SSD right now.

**Comment by TitansBane** [comment]
No, you'll need to clear up some space elsewhere, and it will finish installing. Once completed, you can delete the download folder from Elysium, eliminating 200 GB of space.
